app/models/pawn.rb

```diff
@@ -1,33 +1,74 @@
 class Pawn < Piece
+def valid_move?(new_x, new_y)
 
-  def valid_move?(new_x, new_y)
-
+    #at 5,5 want to move to 5,6 -> return true
     return true if valid_diagonal_move?(new_x, new_y)
-
+    return true if en_passant?(new_x, new_y)
     return false if new_x != x_pos
 
     if white?
-      return false if new_y >= y_pos
-    else
       return false if new_y <= y_pos
+    else
+      return false if new_y >= y_pos

```

Comment:
Double check that the <= and >= are doing what you intended. I don't believe they are both supposed to have the =